docs/faq/authentication.md

```
@@ -0,0 +1,377 @@
+   import jwt
+   
+   token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+   try:
+       decoded = jwt.decode(token, options={"verify_signature": False})
+       print("Token scopes:", decoded.get('scopes', []))
+   except Exception as e:
+       print(f"Error decoding token: {e}")
+   ```
+
+2. **Verify Required Scope**: Check the API documentation for the required scope
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)